cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -144,12 +153,12 @@ public void dismiss() {
         mAnchor.destroyDrawingCache();
         mAnchor.getViewTreeObserver().removeOnPreDrawListener(mPreDrawListener);
         mPopup.dismiss();
-        
```

Comment:
This is the auto modified by Android Studio.
Unless have any changing on the corresponding file, it is hard to avoid this indent changes. fyi.